curriculum/challenges/english/blocks/workshop-media-catalogue/68b99be831fb229c9d950296.md

```diff
@@ -0,0 +1,375 @@
+---
+id: 68b99be831fb229c9d950296
+title: Step 42
+challengeType: 20
+dashedName: step-42
+---
+
+# --description--
+
+Using the same logic, create an `if` statement that runs when the `series` list is not empty. Inside it, concatenate the string `=== TV SERIES ===` (ending with a newline character) to the current value of `result`.
+
+Then, create a `for` loop that uses enumeration to add the items in the `series` list to the `result` variable. As you did previously, start indexing the values from `1`.
```

Comment:
here you asking enumeration, but you did not specify it the first time you asked for the loop inside `__str__`, maybe you want to specify it over there too?